.github/triage-new-issues.yml

```diff
@@ -0,0 +1,7 @@
+# This is based off of reading the actual source code of the bot. :/
+# https://github.com/tunnckoCore/triage-new-issues/blob/master/src/index.js#L7
+#
+# While this file is currently a no-op, it serves the purpose of
+# documenting that this bot is indeed being used, since this is a
+# non-standard probot bot.
+label: "Please triage this"
```

Comment:
I tried to test this bot using #16, but as you can see, it didn't work. Maybe it's because I'm the owner of the repo.